FWCore/ParameterSet/python/Config.py

```diff
@@ -137,6 +138,7 @@ def __init__(self,name,*Mods):
         self.options = Process.defaultOptions_()
         self.maxEvents = Process.defaultMaxEvents_()
         self.maxLuminosityBlocks = Process.defaultMaxLuminosityBlocks_()
+        self.MessageLogger = MessageLogger # intentionally not cloned
```

Comment:
Not cloning here means that in the following
```py
process = cms.Process("Foo")
process.MessageLogger.SomeCategory = cms.untracked.PSet()
process.load('FWCore.MessageService.MessageLogger_cfi')
```
the `process.load()` does not override the settings done before it.

Reply:
Which, I'm assuming, is the desired behavior (to allow independent modifications of the MessageLogger in different packages).